cmd/maya-agent/app/exporter/collector/collector.go

```diff
@@ -164,32 +209,80 @@ func (e *VolumeExporter) collect() error {
 		return err
 	}
 
-	err = json.NewDecoder(resp.Body).Decode(&metrics)
+	err = json.NewDecoder(resp.Body).Decode(&metrics1)
 	if err != nil {
 		log.Printf("could not decode OpenEBS Volume controller metrics: %v", err)
 		return err
 	}
 
-	rIOPS, _ := strconv.ParseFloat(metrics.ReadIOPS, 64)
+	err = json.NewDecoder(resp.Body).Decode(&metrics2)
+	if err != nil {
+		log.Printf("could not decode OpenEBS Volume controller metrics: %v", err)
+		return err
+	}
+
+	iRIOPS, _ := strconv.ParseFloat(metrics1.ReadIOPS, 64)
+	fRIOPS, _ := strconv.ParseFloat(metrics2.ReadIOPS, 64)
+	rIOPS := fRIOPS - iRIOPS
```

Comment:
this operation is being used more then once, better to move in a function and use it.